umake/frameworks/baseinstaller.py

```diff
@@ -173,21 +174,24 @@ def get_metadata_and_check_license(self, result):
             logger.error("An error occurred while downloading {}: {}".format(self.download_page, error_msg))
             UI.return_main_screen()
 
-        url, checksum = (None, None)
-        with StringIO() as license_txt:
+        with StringIO() as license_text:
+            url, checksum = (None, None)
             in_license = False
             in_download = False
+
```

Comment:
For this (and following space lines): they are there for readibility, please do not stash them. I'll comment if there are occurences of some that I think make sense to stash